MapleServer2/PacketHandlers/Game/ClubHandler.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System.Collections.Generic;
+using MaplePacketLib2.Tools;
+using MapleServer2.Constants;
+using MapleServer2.Packets;
+using MapleServer2.Servers.Game;
+using MapleServer2.Types;
+using Microsoft.Extensions.Logging;
+
+namespace MapleServer2.PacketHandlers.Game
+{
+    public class ClubHandler : GamePacketHandler
+    {
+        public override RecvOp OpCode => RecvOp.CLUB;
+
+        public ClubHandler(ILogger<ClubHandler> logger) : base(logger) { }
+
+
+        public override void Handle(GameSession session, PacketReader packet)
+        {
+            byte mode = packet.ReadByte(); //Mode
+
+            switch (mode)
+            {
+                //Club Create
```

Comment:
I personally would prefer to have this as a comment in the line of case 0x1, similar to how it's done in line 20.